packages/azure/data_stream/firewall_logs/elasticsearch/ingest_pipeline/default.yml

```
@@ -192,7 +329,21 @@ processors:
       field:
         - azure.firewall.proto
       ignore_missing: true
-
+  - lowercase:
+      field: azure.firewall.protocol
+      ignore_failure: true
```

```suggestion
  - lowercase:
      field: azure.firewall.protocol
      if: ctx.azure?.firewall?.protocol != null
```
Avoid `ignore_failure` clause.